contrib/examples/cluster-api-controllers-template.yaml

```diff
@@ -48,6 +48,7 @@ objects:
           args:
           - --log-level=debug
           - --default-availability-zone=${DEFAULT_AVAILABILITY_ZONE}
+          - --incluster=false
```

Comment:
How will this work if we want to use cluster-operator in a remote cluster to build out more clusters from there?

Reply:
Oof thats going to be tough. Feels like it should be a cluster specific setting.

Reply:
Nevermind. All of the Machine controllers that we run should use `incluster=false`. We will never be creating a control plane machineset in-cluster, even when a user in the remote cluster is creating more clusters. We should make sure that we are setting this flag in the ClusterAPI controller manager that we start in the remote cluster as well.

Reply:
Per IRC we actually might be, but in any case the incluster flag only blocks deletion of master machines, which we don't expect to exist there, so it seems safe and correct to leave the incluster controllers with this set to the default of true.